airbyte-cdk/python/unit_tests/sources/declarative/test_factory.py

```diff
@@ -41,6 +40,7 @@
 from airbyte_cdk.sources.declarative.stream_slicers.list_stream_slicer import ListStreamSlicer
 from airbyte_cdk.sources.declarative.transformations import AddFields, RemoveFields
 from airbyte_cdk.sources.declarative.transformations.add_fields import AddedFieldDefinition
+from dateutil.relativedelta import relativedelta
```

Comment:
note: i have a PR https://github.com/airbytehq/airbyte/pull/18411 that is going to fix this one since i also modified that file. going to publish and merge this in a few min